simespresso/io_QE/espresso_class.py

```diff
@@ -0,0 +1,815 @@
+import logging
+import os.path
+import subprocess
+import sys
+
+import numpy as np
+from enum import Enum
+from simphony.core.cuba import CUBA
+from simphony.core.data_container import DataContainer
+from simphony.cuds.lattice import Lattice
+from simphony.cuds.particles import Particle, Particles
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class qe_functions(object):
+    '''
+    functions for reading and writing quantum espresso input and output files
+    '''
+
+    def __init__(self):
+        self.SP = DataContainer()  # System Model Equations and Material
+        # relations (Governing Equations)
+        self.SD = DataContainer()  # System Material Description and
+        # State Data including Boundaries (not conditions)
+        self.BC = DataContainer()  # Boundary conditions
+        self.CM = DataContainer()  # Computational Methods
+        # (numerical and solver aspects only)
+        self.pc = Particles('quantum_espresso_particles')
+
+    def read_espresso_output_file(self, file_name):
```

Comment:
I think the comment about the name of this method from @ahashibon (https://github.com/simphony/simphony-espresso-dft/pull/7#discussion_r42113408 ) still needs to be addressed.